common/src/main/java/com/viaversion/viabackwards/protocol/protocol1_13_2to1_14/packets/BlockItemPackets1_14.java

```diff
@@ -64,7 +64,7 @@ public class BlockItemPackets1_14 extends com.viaversion.viabackwards.api.rewrit
     private EnchantmentRewriter enchantmentRewriter;
 
     public BlockItemPackets1_14(Protocol1_13_2To1_14 protocol) {
-        super(protocol);
+        super(protocol, Type.ITEM1_13_2, Type.ITEM1_13_2_SHORT_ARRAY);
```

Comment:
It's unused so not super important, but should technically be ITEM1_13 and ITEM1_13_SHORT_ARRAY

Reply:
Oops wrong class, I meant BlockItemPackets1_13

Reply:
I decided to pass null in order to match VV codestyle in legacy item rewriters